common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -234,75 +267,40 @@ class CapaMixin(ScorableXBlockMixin, CapaFields):
     """
         Core logic for Capa Problem, which can be used by XModules or XBlocks.
     """
-    def __init__(self, *args, **kwargs):
```

Comment:
@cpennington @ormsbee Okay, I have removed `init()` and moved the code to properties.

Reply:
@ormsbee Whenever you are able to get to this, the one area I would like a deeper review on is the seed generation logic.